core/src/main/resources/hudson/model/Label/index.jelly

```diff
@@ -43,8 +43,7 @@ THE SOFTWARE.
 
       <div>
         <h2>${%Nodes}</h2>
-        <j:forEach var="n" items="${it.sortedNodes}">
-          <j:set var="c" value="${app.getComputer(n.nodeName)}"/>
```

Comment:
Note that this was missing a null check on `c`, which Jelly would quietly tolerate. Handled better now with `.filter(Objects::nonNull)`.